Workbooks/Azure Advisor/Cost Optimization/Storage/CostWorkbookStorage.workbook

```diff
@@ -393,7 +499,7 @@
     }
   ],
   "fallbackResourceIds": [

```

Comment:
can we keep "fallbackResourceIds" as azure monitor?

Reply:
@pkrishnanms Done.